docs/policies-and-procedures.md

```diff
@@ -0,0 +1,35 @@
+---
+id: policies-and-procedures
+title: Policies & Procedures
+slug: /policies-and-procedures
+---
+
+## Emergency Contact
+
+If a service is down or an issue needs to be looked at immediately call Christopher Lamm on his phone (number availabe in Slack). Do not text or send a Slack message. If after 10 pm eastern you will need to call once, let it go to voicemail, then call a second time.
+
+## Security
+
+* Setup 2fa for all services that provide it
+* Save all sensitive information in our secrets manager
```

Comment:
"secrets manager" <---I'm guessing you're referring to 1Password. Are we not naming it for a reason?

Reply:
Yes since this is publically available we don't want to reveal too much information like that.

Reply:
isn't this PR public? @theantichris